src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -1295,6 +1295,215 @@ public void PasteWithSelectionInReadonlyArea()
             AssertCaretVirtualPosition(2, 4);             
         }
 
+        [Fact]
+        public void DeleteLineWithOutSelection()
+        {
+            Submit(
+@"1",
+@"1
+");                                                                                                                        
+            var caret = Window.TextView.Caret;                               
+
+            // DeleteLine with caret in readonly area
+            caret.MoveToPreviousCaretPosition();
+            caret.MoveToPreviousCaretPosition();
+            caret.MoveToPreviousCaretPosition();
+
+            AssertCaretVirtualPosition(1, 1);
+            Task.Run(() => Window.Operations.DeleteLine()).PumpingWait();
+            Assert.Equal("> 1\r\n1\r\n> ", Window.TextView.TextBuffer.CurrentSnapshot.GetText());
+            AssertCaretVirtualPosition(1, 1);
+
+            // DeleteLine with caret in active prompt
+            caret.MoveToNextCaretPosition();
+            caret.MoveToNextCaretPosition();
+            caret.MoveToNextCaretPosition();
+            Window.InsertCode("int x");
+            Window.Operations.BreakLine();
+            Window.InsertCode(";");
+            for (int i = 0; i < 11; ++i)
```

Comment:
It's the position in the middle of the first active prompt. I used a loop just to avoid a list of call to `MoveToNextCaretPosition`.